thoth/slo_reporter/sli_references.py

```diff
@@ -46,5 +46,12 @@ def _add_dashbords(start_time_epoch: datetime.datetime, end_time_epoch: datetime
     )
     html_inputs["Thoth SLI/SLO"] = sli_slo_dashboard_url
 
+    sli_slo_dashboard_name = f"thoth-reports"
+    sli_slo_dashboard_url = (
+        f"https://grafana.datahub.redhat.com/dashboard/db/{sli_slo_dashboard_name}?"
+        + f"refresh=1m&orgId=1&from={start_time_epoch}&to={end_time_epoch}"
+    )
+    html_inputs["Thoth Reports"] = sli_slo_dashboard_url
```

Comment:
what are those reports? and description we can provide so that people know why to click the link?

Reply:
Added description for each reference.